docs/reference/supervisor.md

```diff
@@ -1284,6 +1284,8 @@ The last three are respectively the angular velocities around the x, y and z axe
 
 ---
 
+#### `wb_supervisor_node_save_state`
+#### `wb_supervisor_node_reset_state`
```

Comment:
I would call this function `wb_supervisor_node_load_state` so that we better understand it is the counterpart of `_save_state`.